Commands/Gambling/roulette.py

```diff
@@ -0,0 +1,72 @@
+# /commands/basic/help.py
+import discord
+from discord.ext import commands
+from typing import Literal
+import random as r
```

Comment:
Random shouldn't be left as just `r`, instead name it something more readable, like `rand`

Reply:
updated